src/routes/onboarding/set-diet-goal-reminders/set-diet-goals.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Form } from "../../../components"
+
+
+export const SetDietGoals: React.FC = () =>  {
+  return (
+    <div className="bg-[#C9FBC9] h-screen flex flex-col items-center m-auto">
```

Comment:
You should use the classes predefined in tailwind.config.js as much as you can. 
Here specifically, instead of bg-[#C9FBC9], use bg-lightGreen